src/ParallelHelper.Plugin/source.extension.vsixmanifest

```diff
@@ -11,9 +11,24 @@
         <Tags>C#; Parallel; Asynchronous; Concurrency; Bugs; Best Practices; TPL; Task; QC; Static Analysis; async; await</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.3,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.3,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.3,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.3,17.0)" >
+          <ProductArchitecture>x86</ProductArchitecture>
```

Comment:
Is the `ProductArchitecture` node necessary for Visual Studio 2022? Unfortunately, it fails to build with the current installation target. If not, I'd prefer to leave it away.